docs/OCP_CI_Tutorials/Step_Registry/Step_Registry_Ref_Guide.md

```diff
@@ -0,0 +1,167 @@
+# Step Registry - Ref Guide<!-- omit from toc -->
+
+## Table of Contents<!--omit from toc-->
+- [Table of Contents](#table-of-contents)
+- [What is a Ref?](#what-is-a-ref)
+- [How to Create a Ref](#how-to-create-a-ref)
+  - [Preparation](#preparation)
+  - [Populating the Files](#populating-the-files)
+    - [`new-step-ref.yaml`](#new-step-refyaml)
+    - [`new-step-commands.sh`](#new-step-commandssh)
+    - [`OWNERS`](#owners)
+    - [`README.md`](#readmemd)
+  - [Finalization](#finalization)
+
+
+## What is a Ref?
+A ref is the most basic step in the step registry, it can be thought of as the building-blocks of other steps. A ref is essentially a BASH script that be used on it's own or as part of a chain or workflow. A ref is comprised of five files:
```

Comment:
```suggestion
A ref is the most basic step in the step registry, it can be thought of as the building-blocks of other steps. A ref is essentially a BASH script that be used on its own or as part of a chain or workflow. A ref is comprised of five files:
```